libnavigation-sku/src/main/java/com/mapbox/navigation/sku/accounts/MapboxNavigationAccounts.kt

```diff
@@ -48,4 +48,17 @@ internal class MapboxNavigationAccounts private constructor() {
     fun navigationStarted() {
         skuGenerator?.onNavigationStart()
     }
+
+    fun buildResourceUrlWithSku(resourceUrl: String, querySize: Int): String {
+        val urlBuilder = StringBuilder(resourceUrl)
+        if (querySize == 0) {
+            urlBuilder.append("?")
+        } else {
+            urlBuilder.append("&")
+        }
+
+        val skuToken = obtainSkuToken()
+        urlBuilder.append("sku=$skuToken")
```

Comment:
Hardcoded `sku`

Reply:
You have in mind replacing with a constant value?